README.md

```diff
@@ -149,7 +152,8 @@ JSON example:
 ```json
 "OpenTok": {
 	"apiKey": "<key>",
-	"apiSecret": "<secret>",
+  "apiSecret": "<secret>",
```

Comment:
the indentation is a bit off here
https://github.com/jeffswartz/OpenTokRTC-V2/tree/configurable-opentok-js-src#opentok-configuration